internal/web3ext/web3ext.go

```diff
@@ -881,6 +881,12 @@ web3._extend({
 			params: 2,
 			inputFormatter: [null, function (val) { return !!val; }]
 		}),
+		new web3._extend.Method({
+			name: 'GetBlockByNumber',
```

Comment:
should it be 
```suggestion
			name: 'getBlockByNumber',
```

like other lines?